nextcord/interactions.py

```diff
@@ -1367,3 +1375,61 @@ def __init__(
     ) -> None:
         super().__init__(state=state, channel=channel, data=data)
         self._interaction: Interaction = interaction
+
+
+class ApplicationCommandPermissions:
+    """Represents an application command's permissions.
+
+    Parameters
+    ----------
+    id: :class:`int`
+        The ID of the role, user, or channel that this permission overwrite is for.
+    type: :class:`ApplicationCommandPermissionType`
+        The type of object that the permission overwrite is for.
+
+        Can be for a role, user, or channel.
+    permission: :class:`bool`
+        Whether the permission overwrite is disallowing or allowing usage of the command.
+
+    .. versionadded:: 3.0
```

Comment:
versionadded under description and above headers like parameters/attributes please (for both)